friendly_traceback/core.py

```diff
@@ -429,31 +430,11 @@ def cleanup_tracebacks(records, write_err):
         they could possibly invoke some code from files which we had
         decided to exclude.
     """
-    if not records:
-        return records  # likely an empty list
-
-    for begin, record in enumerate(records):
-        frame, filename, linenumber, _func, lines, index = record
-        if is_excluded_file(filename):
-            continue
-        else:
-            break
-
+    records = list(dropwhile(lambda record: is_excluded_file(record.filename), records))
```

Comment:
I have never used itertools - I really need to do so. It does simplify the code quite a bit here.